src/main/java/jenkins/advancedqueue/sorter/SorterStrategyDescriptor.java

```diff
@@ -28,7 +28,6 @@
 /**
  * Implements descriptor for {@link SorterStrategy}.
  * 
- * @author Oleg Nenashev <nenashev@synopsys.com>, Synopsys Inc.
```

Comment:
this was breaking for invalid 'html', when I went to check the correct syntax the top 2 articles just recommended removing them, e.g. https://www.vojtechruzicka.com/stop-using-javadoc-author-tag/

any objection @oleg-nenashev